actors.md

```diff
@@ -708,3 +708,233 @@ func UpdatePeerID(pid PeerID) {
 ## Payment Channel Broker Actor
 
 TODO
+
+## Multisig Account Actor
+
+A basic multisig account actor. Allows sending of messages like a normal account actor, but with the requirement of M of N parties agreeing to the operation. Any party can veto any operation (which is a potentially debateable design decision). Completed and/or cancelled operations stick around in the actors state until explicitly cleared out.
```

Comment:
technically "any party can veto" is not correct. If you pick n of m and n approve and one veto, transaction gets exectured.

Reply:
lets go with "you can cancel your own transaction, or the transactions of anyone who is no longer a signer", and/or "any transaction that hasnt been approved in X blocks can be removed"